config.c

```diff
@@ -130,8 +130,31 @@ void read_FTLconf(void)
 	else
 		logg("   RESOLVE_IPV4: Don\'t resolve IPv4 addresses");
 
+	// DBFILE
+	// defaults to: "/etc/pihole/pihole-FTL.db"
+	buffer = parse_FTLconf(fp, "DBFILE");
+
+	errno = 0;
+	if(!(buffer != NULL && sscanf(buffer, "%127ms", &FTLfiles.db)))
+	{
+		// Use standard path if no custom path was obtained from the config file
+		FTLfiles.db = strdup("/etc/pihole/pihole-FTL.db");
+	}
+
+	// Test if memory allocation was successful
+	if(FTLfiles.db == NULL && errno != 0)
+	{
+		logg("FATAL: Allocating memory for FTLfiles.db failed (%s, %i). Exiting.", strerror(errno), errno);
+		exit(EXIT_FAILURE);
+	}
+	else if(FTLfiles.db != NULL && strlen(FTLfiles.db) > 0)
+		logg("   DBFILE: Using %s", FTLfiles.db);
+	else
+		logg("   DBFILE: Not using database due to empty filename");
```

Comment:
I don't see where `FTLfiles.db` is being set to the user's input?

Reply:
I added a comment in 9c7bac7